utt/libutt/src/api/mint.cpp

```diff
@@ -6,10 +6,30 @@
 #include <utt/Coin.h>
 #include <utt/Serialization.h>
 
+std::ostream& operator<<(std::ostream& out, const libutt::api::operations::Mint& mint) {
+  out << *(mint.op_);
+  return out;
+}
+std::istream& operator>>(std::istream& in, libutt::api::operations::Mint& mint) {
+  in >> *(mint.op_);
+  return in;
+}
+
 namespace libutt::api::operations {
 Mint::Mint(const std::string& uniqueHash, size_t value, const std::string& recipPID) {
   op_.reset(new libutt::MintOp(uniqueHash, value, recipPID));
 }
+Mint::Mint() { op_.reset(new libutt::MintOp()); }
+Mint::Mint(const Mint& other) {
```

Comment:
When do we copy the classes that you added copy operator for them?

Reply:
Currently, we don't. I have added a copy construct as part of the paradigm that assignment operator and copy constructor always come together